parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -85,6 +89,8 @@ pub mod pallet {
 		#[pallet::constant]
 		type GatewayAddress: Get<H160>;
 
+		type MessageConverter: ConvertMessage;
```

Comment:
Needs a documentation comment.

```suggestion
		/// Convert inbound message to XCM
		type MessageConverter: ConvertMessage;
```